Source/WebKit/WebProcess/WebCoreSupport/WebFrameLoaderClient.cpp

```diff
@@ -984,6 +986,15 @@ void WebFrameLoaderClient::dispatchDecidePolicyForNavigationAction(const Navigat
     navigationActionData.lockHistory = navigationAction.lockHistory();
     navigationActionData.lockBackForwardList = navigationAction.lockBackForwardList();
     navigationActionData.privateClickMeasurement = navigationAction.privateClickMeasurement();
+#if PLATFORM(MAC) || HAVE(UIKIT_WITH_MOUSE_SUPPORT)
+    if (auto mouseEventData = navigationAction.mouseEventData()) {
```

Comment:
Do we need this in WebFrameLoaderClient::dispatchDecidePolicyForNewWindowAction too?

Reply:
Good catch, yes we do.